trieste/acquisition/rule.py

```diff
@@ -288,8 +295,15 @@ def acquire(
                 if summary_writer:
                     with summary_writer.as_default(step=step_number):
                         batched_points = tf.expand_dims(chosen_point, axis=0)
-                        value = self._acquisition_function(batched_points)[0][0]
-                        logging.scalar(f"EGO.acquisition_function/maximum_found[{i+1}]", value)
+                        values = self._acquisition_function(batched_points)[0]
+                        if len(values) == 1:
+                            logging.scalar(
+                                f"EGO.acquisition_function/maximum_found[{i + 1}]", values[0]
+                            )
+                        else:  # vectorized acquisition function
```

Comment:
this is a very corner case where we use both greedy and multipoint, in theory why not but not sure this is actually feasible at the moment? this extra bit of code doesn't hurt though

Reply:
yeah, this can't happen at the moment, but feels like it's safer to leave this in so if it does become possible we don't forget to update it.